test/production/app-dir/mangle-reserved/mangle-reserved.test.ts

```diff
@@ -6,9 +6,11 @@ createNextDescribe(
     files: __dirname,
   },
   ({ next, isTurbopack }) => {
-    ;(isTurbopack ? it.skip : it)('should work using cheerio', async () => {
-      const $ = await next.render$('/')
-      expect($('p').text()).toBe('AbortSignal')
+    it('should work using cheerio', async () => {
```

Comment:
`it.skip` might be best if it means that the test runner reports that the test was skipped, versus just silently skipping it while marking it as passed.

Reply:
Thanks. I did this because a lint rule failed, but I'll try disabling lint

Reply:
Did it